vix/VixHost.py

```diff
@@ -94,29 +98,58 @@ def __init__(self, service_provider=VIX_SERVICEPROVIDER_DEFAULT, host=None, cred
 
         assert self._handle == None, 'Instance is already connected.'
 
+        if service_provider in (self.VIX_SERVICEPROVIDER_VMWARE_VI_SERVER, self.VIX_SERVICEPROVIDER_VMWARE_WORKSTATION_SHARED):
+            if isinstance(host, collections.Sequence) and len(host) == 2 and host[1] != 0:
+                host = ('%s:%s' % host, 0)
+
+            elif host is None and service_provider == self.VIX_SERVICEPROVIDER_VMWARE_WORKSTATION_SHARED:
+                if platform.system() == 'Windows':
+                    vmhostd_proxy_xml = os.path.join(os.getenv('ProgramData'), 'VMware', 'hostd', 'proxy.xml')
+                elif platform.system() == 'Linux':
+                    vmhostd_proxy_xml = '/etc/vmware/hostd/proxy.xml'
+                else:
+                    raise NotImplemented('Unrecognized OS or architecure ({0}, {1}, {2})'.format(platform.architecure(), platform.machine(), platform.system()))
```

Comment:
I don't have macOS system. But there is no feature like *shared virtual machines* in VMware Fusion.

In the [VMware Fusion 8 Documentation](https://pubs.vmware.com/fusion-8/index.jsp?topic=%2Fcom.vmware.fusion.using.doc%2FGUID-30FCA4B3-D9FD-40AF-8817-F0902AE6D758.html):

> Shared virtual machines in Fusion work differently than shared virtual machines in Workstation Pro. In Workstation Pro, a shared virtual machine is a virtual machine on the host system that remote Workstation Pro users can access as a remote virtual machine. In Fusion, shared virtual machines can be accessed only on the local Mac host.